content/efcore/getting-started/querying.md

```diff
@@ -0,0 +1,91 @@
+# Querying Data 
+ 
+In this lesson, you'll learn how to use querying to fetch one or more items from your database. EF Core uses LINQ to query data, so it is important that you understand LINQ before going through this lesson. Check out the [LINQ lesson](../../csharp/getting-started/linq) in the C# Interactive Tutorial if you need a refresher. 
```

Comment:
The link to the LINQ lesson is broken. I think it is just missing the file extension (linq.md)